src/Extracting/ParsesValidationRules.php

```diff
@@ -145,9 +145,14 @@ protected function normaliseRules(array $rules): array
         // Now this will return the complete ruleset.
         // Nested array parameters will be present, with '*' replaced by '0'
         $newRules = Validator::make($testData, $rules)->getRules();
-
-        // Transform the key names back from 'ids.0' to 'ids.*'
+       
         return collect($newRules)->mapWithKeys(function ($val, $paramName) use ($rules) {
+            // Transform the key names back from '__asterisk__' to '*'
+            if (Str::contains($paramName, '__asterisk__')) {
```

Comment:
Hi, where does the `__asterisk__` come from? I can't find that anywhere in the codebase.

Reply:
Ah, never mind; I see, it's from the framework.